vscode/src/azure/commands.ts

```diff
@@ -289,7 +289,7 @@ export async function initAzureWorkspaces(context: vscode.ExtensionContext) {
           if (file) {
             const doc = await vscode.workspace.openTextDocument({
               content: file,
-              language: "plaintext",
+              language: "json",
```

Comment:
I believe this is the right value based on strings from the selector UI, but as @minestarks mentioned we will merge once we have more manual validation done.